src/services/product.service.js

```diff
@@ -0,0 +1,30 @@
+const { productModel } = require("../models");
+
+const getProductList = async (userId) => {
+  // if(page==내림차순){ , page, sort, category
+
+  // }
+
+  // if(category){
+
+  // }
+
+  // if(sort){
+
+  // }
+
+  const product = await productModel.productList(userId);
+
+  return product;
+};
+
+const getTotalProduct = async (req, res) => {
+  const product = await productModel.totalProduct(req, res);
+
+  return product;
+};
```

Comment:
```suggestion
const getTotalProduct = async () => {
  const product = await productModel.totalProduct();

  return product;
};
```
`req`와 `res`는 요청과 응답 객체로, contorller layer에서만 이용되어야 하는 매개변수입니다. controller 단에서 요청/응답 처리가 모두 마무리 되었다면, service layer에서는 해당하는 객체를 활용해서도 안되고, model 레이어로 넘겨주어서도 안됩니다 :) 

request와 response를 관린하는 역할은 controller에만 적용하여 SOC(Seperation Of Concern) 를 지켜주세요